Resources/Help/analyses/ttestbayesianpairedsamples_nl.md

```diff
@@ -62,8 +62,8 @@ Met de gepaarde t-toets kan de gebruiker de effectgrootte schatten en de nulhypo
   - BF0+: De Bayes factor die bewijs geeft voor de nulhypothese, ten opzichte van de eenzijdige alternatieve hypothese dat het verschil groter is dan 0.
   - BF0-: De Bayes factor die bewijs geeft voor de nulhypothese, ten opzichte van de eenzijdige alternatieve hypothese dat het verschil kleiner is dan 0.
 - error %: De fout van de Gaussiaanse kwadratuur intergratie methode die wordt gebruikt op de Bayes factor te berekenen.
-- W: The toets statistiek van de rangtekentoets.
-- Rhat: Indicatie van convergentie voor de rangtekentoets. De ratio van variantie binnen elke MCMC keten, en de totale variantie van de MCMC ketens voor de delta parameter. Waarden gelijk aan 1 duiden op convergentie.
+- W: De toets statistiek van de rangtekentoets.
+- Rhat: Indicatie van convergentie voor de rangtekentoets. Een ratio die de varianties vergelijkt binnen en tussen de MCMC ketens voor de delta parameter. Waarden minder dan, of gelijk aan, 1 duiden op convergentie.
```

Comment:
R-dakje <- Tim's schuld

Misschien is "Indicatie" te direct overgenomen en moet het 

"Mate van convergentie voor de MCMC procedure."

Omdat de toets zelf niet perse convergeert.

Reply:
Misschien "Index van convergentie voor de MCMC procedure"